### PR TITLE
tester: differentiate expected output by golang version

### LINKED ISSUE #3082: Test failures with Golang 1.16 beta1
With Docker 1.16-beta1, some tests in package `github.com/open-policy-agent/opa/tester` are failing:

```
=== RUN   TestPrettyReporterVerboseBenchmark
    reporter_test.go:380: Expected:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123 ns/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
        
        
        Got:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123.0 ns/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
--- FAIL: TestPrettyReporterVerboseBenchmark (0.00s)
=== RUN   TestPrettyReporterVerboseBenchmarkShowAllocations
    reporter_test.go:457: Expected:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
        
        
        Got:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        data.foo.bar.test_baz	    1000	       123.0 ns/op	       123.0 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
--- FAIL: TestPrettyReporterVerboseBenchmarkShowAllocations (0.00s)
=== RUN   TestPrettyReporterVerboseBenchmarkShowAllocationsGoBenchFormat
    reporter_test.go:535: Expected:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        BenchmarkDataFooBarTestBaz	    1000	       123 ns/op	       123 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
        
        
        Got:
        
        FAILURES
        --------------------------------------------------------------------------------
        data.foo.bar.test_corge: FAIL (0s)
        
        
        SUMMARY
        --------------------------------------------------------------------------------
        BenchmarkDataFooBarTestBaz	    1000	       123.0 ns/op	       123.0 timer_rego_query_eval_ns/op	      91 B/op	       0 allocs/op
        data.foo.bar.test_qux: ERROR (0s)
          some err
        data.foo.bar.test_corge: FAIL (0s)
        --------------------------------------------------------------------------------
        PASS: 1/3
        FAIL: 1/3
        ERROR: 1/3
--- FAIL: TestPrettyReporterVerboseBenchmarkShowAllocationsGoBenchFormat (0.00s)
```

The failure is due to a formatting change: the time per operation is displayed `123 ns/op` in Go 1.15, and `123.0 ns/op` in Go 1.16-beta1.

### PR DESCRIPTION
**Pull request by srenatus**
Fixes #3082.

### CODE REVIEW

File: tester/reporter_test.go [hunk 1]
```diff
@@ -436,23 +420,7 @@ func TestPrettyReporterVerboseBenchmarkShowAllocations(t *testing.T) {
...
-data.foo.bar.test_qux: ERROR (0s)
-  some err
-data.foo.bar.test_corge: FAIL (0s)
---------------------------------------------------------------------------------
-PASS: 1/3
-FAIL: 1/3
-ERROR: 1/3
-`
-
+        exp := fixtureReporterVerboseBenchmarkShowAllocations
```
Comment (olivierlemasle) [inline tester/reporter_test.go:423]: Linter failure: wrong indentation

**Review by olivierlemasle — APPROVED** [review]
Works with Go 1.16-beta1

**Comment by srenatus** [comment]
@olivierlemasle thanks for checking! 👏

**Review by anderseknert — APPROVED** [review]

**Review by tsandall — APPROVED** [review]
LGTM

**Pull request merged**